lib/matplotlib/offsetbox.py

```diff
@@ -1205,6 +1225,7 @@ def __init__(self, arr,
                  **kwargs
                  ):
 
+        OffsetBox.__init__(self)
```

Comment:
As I recall the terrifying thing in mplot3d is the class promotion which I don't think this will have an impact on.